src/Application.php

```diff
@@ -37,15 +37,7 @@ public function bootstrap()
         // Call parent to load bootstrap from files.
         parent::bootstrap();
 
-        if (PHP_SAPI === 'cli') {
```

Comment:
I would prefer if we retain the SAPI check here and avoid method call when not needed.